flink-formats/flink-avro/src/main/java/org/apache/flink/formats/avro/typeutils/AvroSchemaConverter.java

```diff
@@ -272,11 +276,14 @@ private static DataType convertToDataType(Schema schema) {
 				return DataTypes.TIMESTAMP(3)
 						.bridgedTo(java.sql.Timestamp.class)
 						.notNull();
-			}
-			if (schema.getLogicalType() == LogicalTypes.timestampMicros()) {
+			} else if (schema.getLogicalType() == LogicalTypes.timestampMicros()) {
 				return DataTypes.TIMESTAMP(6)
 						.bridgedTo(java.sql.Timestamp.class)
 						.notNull();
+			} else if (schema.getLogicalType() == LogicalTypes.timeMicros()) {
+				return DataTypes.TIME(6)
+					.bridgedTo(LocalTime.class)
```

Comment:
Shouldn't this be a SQL type like the others? Is this method actually used?

Reply:
Actually, those should be the default conversion classes from `java.time`. I changed that. It is not used, at least for now. We might need that later e.g. in schema-registry where we will need to convert schema retrieved from schema-registry to a SQL schema. 

It was added as a  complementary method to `org.apache.flink.formats.avro.typeutils.AvroSchemaConverter#convertToSchema(org.apache.flink.table.types.logical.LogicalType)`